src/features/src/changeVolumeFeature.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import Slider from '../../clui-ui-components/Slider'
+import List from '../../clui-ui-components/List'
+
+import FFmpegFeature from '../FFmpegFeature'
+
+type ChangeVolumeConfig = {
+  VOLUME: { value: number; [name: string]: any }
+  [name: string]: any
+}
+
+class ChangeVolumeFeature extends FFmpegFeature {
+  configuration: ChangeVolumeConfig
+
+  constructor(configuration: ChangeVolumeConfig) {
+    super()
+    this.configuration = configuration
+    const { volumeValue } = this.parseConfiguration()
+    const command = this.changeVolumeCommand(volumeValue)
+    this.setFFmpegCommands(command)
+    this.setProgress()
+    this.setFileConfig()
+  }
+
+  parseConfiguration = () => {
+    const { VOLUME } = this.configuration
+    const { VALUE } = this.configuration
```

Comment:
This bug is two-fold here first, the first `VOLUME` is what contains the inner object, there is no `VALUE` inside `this.configuration`. Second is a typo regardless of that `const {value} = VOLUME` would have worked because the key name is not capital

```js
VOLUME: { value: number; [name: string]: any }
```